hedera-node/src/main/java/com/hedera/services/contracts/operation/HederaCallCodeOperation.java

```diff
@@ -89,9 +88,4 @@ public OperationResult execute(final MessageFrame frame, final EVM evm) {
                 addressValidator,
                 precompiledContractMap);
     }
-
-    @Override
-    protected long gas(final MessageFrame frame) {
```

Comment:
Q: Should we add those test cases in now for these paths? 
The ones that got removed when these functions were added.